pkgs/applications/misc/mattercontrol/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, mono, dpkg, makeWrapper, sqlite }:
+
+stdenv.mkDerivation rec {
+  name = "MatterControl-${version}";
+  version = "1.7.5";
+  src = fetchurl {
+    url = "https://mattercontrol.appspot.com/downloads/mattercontrol-linux/release";
```

Comment:
That is not a stable URL so your expected SHA hash won't match once a new version is released upstream.